src/app/shared/components/line-item/line-item-edit-dialog/line-item-edit-dialog.component.ts

```diff
@@ -89,6 +90,7 @@ export class LineItemEditDialogComponent implements OnInit, OnDestroy, OnChanges
     this.product$ = this.shoppingFacade.product$(this.sku$, ProductCompletenessLevel.List);
 
     this.variationOptions$ = this.shoppingFacade.productVariationOptions$(this.sku$);
+    this.variationOptions$.pipe(takeUntil(this.destroy$)).subscribe(x => (this.variationOptions = x ? [...x] : x)); // trigger change detection
```

Comment:
For my understanding: Why is this necessary? What does this fix?

Reply:
in case of redirect to itself (found by findPossibleVariationForSelection), the component lifecycle does not work. (reset selection), so i have to be sure new inputs with same values triggers ngchanges. 

(i dont know, is there a copy operator or somethime in rxjs)